athenian/database.go

```diff
@@ -17,3 +17,16 @@ var ErrNotFound = dbs.ErrNotFound
 type Account = types.Account
 
 type Database = types.Database
+
+type InstaflowStatus = types.InstaflowStatus
+type InstallStatus = types.InstallStatus
+
+const (
+	InstallStatusAccCreated          = types.InstallStatusAccCreated
```

Comment:
That's a lot of aliases. Lets remove the aliases to these enum values (let the caller import `types`), but keep the `InstallStatus` alias.

Reply:
if I understood correct - updated